web/src/actions/apd.js

```diff
@@ -80,7 +80,6 @@ export const fetchApd = () => dispatch => {
     .then(req => {
       const apd = Array.isArray(req.data) ? req.data : null;
       dispatch(receiveApd(apd));
-      dispatch(updateBudget());
```

Comment:
Don't need to update the budget when APDs are *fetched*, but only when one is selected.